homeassistant/components/playstation_network/coordinator.py

```diff
@@ -34,23 +44,39 @@ def __init__(
         hass: HomeAssistant,
         psn: PlaystationNetwork,
         config_entry: PlaystationNetworkConfigEntry,
+        update_interval: timedelta,
     ) -> None:
         """Initialize the Coordinator."""
         super().__init__(
             hass,
             name=DOMAIN,
             logger=_LOGGER,
             config_entry=config_entry,
-            update_interval=timedelta(seconds=30),
+            update_interval=update_interval,
         )
```

Comment:
you could also define a `_update_interval` on the class and just pass `self._update_interval` here. Saves you creating constructors